flytekit/models/sagemaker/training_job.py

```diff
@@ -43,6 +58,15 @@ def volume_size_in_gb(self) -> int:
         """
         return self._volume_size_in_gb
 
+    @property
+    def distributed_protocol(self) -> int:
```

Comment:
Can you add this property to the unit test for the model as well please?

Reply:
Added.